core/controllers/reader.py

```diff
@@ -294,6 +298,68 @@ def get(self, exploration_id):
         self.render_json(self.values)
 
 
+class StorePlaythroughHandler(base.BaseHandler):
+    """Handles a useful playthrough coming in from the frontend to store it."""
+
+    @acl_decorators.can_play_exploration
+    def post(self, exploration_id):
+        """Handles POST requests.
```

Comment:
Please also document what happens to the POST requests, e.g., "appends to existing list of playthroughs or deletes it if already full." It should give me an idea of what the code will do so it's easy to review when I actually read over the code (both today and in the future).